src/lib/plugin_apis/mdraid.api

```diff
@@ -20,6 +20,20 @@ typedef enum {
 #define BD_MD_TYPE_EXAMINEDATA (bd_md_examine_data_get_type ())
 GType bd_md_examine_data_get_type();
 
+/**
+ * BDMDExamineData:
+ * @device: path of the MD device
+ * @level: RAID level of the device
+ * @num_devices: number of devices used by the MD device
+ * @name: name of the MD device
+ * @size: size of the MD device
+ * @uuid: array UUID
+ * @update_time: update time of the MD device
+ * @dev_uuid: UUID of the MD device
```

Comment:
You could say something like `UUID of the member device` here to make it a bit clearer.